crates/quinn_test_client/src/main.rs

```diff
@@ -0,0 +1,215 @@
+use {
+    anyhow::bail,
+    clap::Parser,
+    libp2p_identity::{Keypair, ed25519},
+    quinn_proto::{ConnectionStats, FrameStats, PathStats, UdpStats},
+    std::{net::SocketAddrV4, str::FromStr, time::Duration},
+    tokio::task::{self, JoinSet},
+    tracing::{info, warn},
+    wcn_rpc::{Client, PeerId, client, transport::Priority},
+    wcn_storage_api::{Namespace, RecordVersion, StorageApi, operation::Del, rpc::DatabaseApi},
+};
+
+#[derive(Clone, Parser)]
+struct Config {
+    #[arg(short = 'c', long, default_value_t = 10)]
+    connection_timeout_secs: u64,
+
+    #[arg(short = 'M', long, default_value_t = 1000)]
+    max_concurrent_rpcs: u32,
+
+    #[arg(short = 'i', long, default_value_t = 10)]
+    max_idle_connection_timeout_secs: u64,
```

Comment:
Why are these config values in seconds? If we want to match our prod environment, we should be able to configure them with millisecond precision.